package.json

```diff
@@ -82,19 +78,15 @@
     "fuse-concat-with-sourcemaps": "^1.0.5",
     "glob": "^7.1.1",
     "ieee754": "^1.1.8",
-    "inquirer": "^3.0.6",
     "lego-api": "^1.0.7",
     "mustache": "^2.3.0",
     "postcss": "^6.0.1",
     "pretty-time": "^0.2.0",
     "prettysize": "0.0.3",
     "realm-utils": "^1.0.9",
-    "regexpu-core": "^4.1.3",
-    "request": "^2.79.0",
```

Comment:
We need request, Quantum uses it with one option

Reply:
Hump i don't find the usage in another place than test

Reply:
https://github.com/fuse-box/fuse-box/blob/8aa160b142e6a3d069b192aa19b271e47e329d2a/modules/fuse-box-responsive-api/index.js#L187-L194

Reply:
ops, ok i rewrite commit history for fix that